arwen/contexts/instanceBuilder.go

```diff
@@ -0,0 +1,22 @@
+package contexts
```

Comment:
Missing comments for the exported functions in this file.

Reply:
Added.